crates/bevy_text/src/lib.rs

```diff
@@ -101,6 +107,13 @@ impl Plugin for TextPlugin {
                         // will never modify a pre-existing `Image` asset.
                         .ambiguous_with(CameraUpdateSystem),
                     remove_dropped_font_atlas_sets,
+                    check_visibility::<WithText>
```

Comment:
Is this necessary if we're already doing a `check_visibility` on it with the `SpriteSource` component?

Reply:
Good point, it isn't necessary. I removed it.